apisix/plugins/clickhouse-logger.lua

```diff
@@ -31,7 +32,7 @@ local batch_processor_manager = bp_manager_mod.new(plugin_name)
 local schema = {
     type = "object",
     properties = {
-        endpoint_addr = core.schema.uri_def,
```

Comment:
Would be better to keep the old conf and mark it as deprecated?
Like https://github.com/apache/apisix/blob/1771c51f94d1c5c4f6542a9d90c7c1d777b992d2/apisix/stream/plugins/mqtt-proxy.lua#L40